src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+package mate.academy.dao;
```

Comment:
for the future, better create subpackages for implementations
```suggestion
package mate.academy.dao.impl;
```